src/states/bootstrapping_peer.rs

```diff
@@ -299,21 +303,44 @@ impl Base for BootstrappingPeer {
                 self.reconnect_to_new_section(new_conn_infos);
                 Ok(Transition::Stay)
             }
+            Variant::Bounce { message, .. } => {
+                let sender = msg.src.to_sender_node(sender)?;
+
+                trace!(
+                    "{} - Received Bounce of {:?} from {}. Resending",
+                    self,
+                    MessageHash::from_bytes(&message),
+                    sender,
+                );
+                self.send_message_to_target_later(sender.peer_addr(), message, BOUNCE_RESEND_DELAY);
```

Comment:
So consider that we bootstrap to the network but the node we bootstrap to is currently undergoing relocation and so can't process our `BootstrapRequest` and bounces is back to us. We must resend it because this might be the only bootstrap contact we have and otherwise we would get stuck.

Reply:
I think in this case, a more reasonable and quicker solution shall be: try another not to bootstrap to or just restart bootstrap totally?
A node under relocation is not a good candidate to bootstrap to, as many info (id, section info ..) may change. which will complicate the bouncing procedure, if need to be catered by bouncing?